utp-testing/docker/circleci/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:bullseye-slim
+FROM ubuntu:22.04
```

Comment:
Why the change here?

Reply:
There are some issues with the Debian slim image when I try to run the test app container locally. Switching to the latest ubuntu LTS works like a charm.